tests/libinfra/certificates.go

```diff
@@ -40,3 +49,40 @@ func ContainsCrt(bundle []byte, containedCrt []byte) bool {
 	}
 	return attached
 }
+
+func GetBundleFromConfigMap(ctx context.Context, configMapName string) ([]byte, []*x509.Certificate) {
+	virtClient := kubevirt.Client()
+	configMap, err := virtClient.CoreV1().ConfigMaps(flags.KubeVirtInstallNamespace).Get(ctx, configMapName, v1.GetOptions{})
+	gomega.Expect(err).ToNot(gomega.HaveOccurred())
+	if rawBundle, ok := configMap.Data[components.CABundleKey]; ok {
+		crts, err := cert.ParseCertsPEM([]byte(rawBundle))
+		gomega.Expect(err).ToNot(gomega.HaveOccurred())
+		return []byte(rawBundle), crts
+	}
+	return nil, nil
+}
+
+// EnsurePodsCertIsSynced waits until new certificates are rolled out to all pods
+// that are matching the specified labelselector.
+// Once all certificates are in sync, the final secret is returned
+func EnsurePodsCertIsSynced(labelSelector string, namespace string, port string) []byte {
+	var certs [][]byte
+	gomega.EventuallyWithOffset(1, func() bool {
+		var err error
+		certs, err = libpod.GetCertsForPods(labelSelector, namespace, port)
+		gomega.Expect(err).ToNot(gomega.HaveOccurred())
```

Comment:
Regular `Expect` failure, will fail the whole `Eventually`. Is that what we want here? If not, please change:
```suggestion
	gomega.EventuallyWithOffset(1, func(g gomega.Gomega) bool {
		var err error
		certs, err = libpod.GetCertsForPods(labelSelector, namespace, port)
		g.Expect(err).ToNot(gomega.HaveOccurred())
```

Reply:
I agree. Fixed